src/Features/LanguageServer/Protocol/Handler/FoldingRanges/FoldingRangesHandler.cs

```diff
@@ -58,11 +58,17 @@ public async Task<FoldingRange[]> HandleRequestAsync(FoldingRangeParams request,
 
                 var linePositionSpan = text.Lines.GetLinePositionSpan(span.TextSpan);
 
+                // Filter out single line spans.
+                if (linePositionSpan.Start.Line == linePositionSpan.End.Line)
```

Comment:
I added this check to match local Roslyn, which [filters out single-line regions](http://sourceroslyn.io/#Microsoft.CodeAnalysis.EditorFeatures.Wpf/Structure/AbstractStructureTaggerProvider.cs,224).